modules/persistent-storage-fibre-provisioning.adoc

```diff
@@ -32,11 +32,13 @@ spec:
   accessModes:
     - ReadWriteOnce
   fc:
-    targetWWNs: ['500a0981891b8dc5', '500a0981991b8dc5'] <1>
+    wwids: [scsi-3600508b400105e210000900000490000] <1>
+    targetWWNs: ['500a0981891b8dc5', '500a0981991b8dc5'] <2>
     lun: 2
     fsType: ext4
 ----
-<1> Fibre Channel WWNs are identified as
+<1> World wide identifiers (WWIDs). Either FC `*wwids*` or a combination of FC `*targetWWNs*` and `*lun*` must be set, but not both simultaneously. The FC WWID identifier is recommended over the WWNs target because it is guaranteed to be unique for every storage device, and independent of the path that is used to access the device. The WWID identifier can be obtained by issuing a SCSI Inquiry to retrieve the Device Identification Vital Product Data (`*page 0x83*`) or Unit Serial Number (`*page 0x80*`). FC WWIDs are identified as `*/dev/disk/by-id/*` to reference the data on the disk, even if the path to the device changes and even when accessing the device from different systems.
```

Comment:
`lun: 2` should have `<2>` too, it's either `wwids:` or (`targetWWNs` and `lun`).

Reply:
@jsafrane I have fixed it. Please check

Reply:
We sometimes allow a callout number to be used more than once, but that's only when it's describing an item that repeats.